_posts/2022-12-01-trino-summit-2022-sk-telecom-recap.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Journey to Iceberg with Trino"
+author: "JaeChang Song, Jennifer Oh, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/apple.jpg
+---
+
+This post comes from [the later half of Trino Summit 2022 session]({% post_url 
+2022-11-21-trino-summit-2022-recap %}). Our friends JaeChang and Jennifer from
+SK Telecom, traveled across the globe from South Korea to join us in person! SK
+Telecom recently had some issues scaling Trino on the Hive model, among other
+issues that come with Hive. While some initial tweaking helped speed things up,
+it ultimately never solved the problem. SK Telecom initially tested performance
+after moving on Iceberg with some very impressive results. They talk about their
+journey to Iceberg via Trino.
```

Comment:
Suggest rewriting to "In this talk, Jennifer and JaeChang describe their journey to Iceberg with Trino."